controllers/storagecluster/cephcluster.go

```diff
@@ -708,6 +708,14 @@ func getMonCount(nodeCount int, arbiter bool) int {
 		}
 	}
 
+	if sc != nil && sc.Spec.ManagedResources.CephCluster.MonCount != 0 {
+		if sc.Spec.ManagedResources.CephCluster.MonCount != 3 && sc.Spec.ManagedResources.CephCluster.MonCount != 5 {
+			log.Error(fmt.Errorf("Provided incorrect mon count"), "It is recommended to set mon count as 3 or 5 only")
```

Comment:
You can add this check as kubebuilder tag in the CR?

Reply:
Ah,right. that could be done.
I will update it.

Reply:
Done